### PR TITLE
Added Spanish translation for linux-requirements.md

### PR DESCRIPTION
**Pull request by NicoC0**
### Description
I create the document linux-requirements to the folder 'es' for the translation of the corresponding section in the wiki.
### Related Issue

https://github.com/azerothcore/wiki/issues/539

### CODE REVIEW

File: docs/es/linux-requirements.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+| |
+| :- |
+| MySQL ≥ 5.7.0 |
+| Boost ≥ 1.67 |
+| CMake ≥ 3.16 |
+| Clang ≥ [10](https://github.com/azerothcore/azerothcore-wotlk/actions?query=workflow%3Acore-build) |
+
+#### Ubuntu con MariaDB 10.x
+
+AzerothCore no soporte las versiones 10.6 y 10.5 de MariaDB.
```
Comment (Tony931023) [inline docs/es/linux-requirements.md:17]:
````suggestion
AzerothCore no soporta las versiones 10.6 y 10.5 de MariaDB.
```

File: docs/es/linux-requirements.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+#### Verificar la versión de cmake
+
+```sh
+cmake --version
+```
+
+La versión de `cmake` **DEBE** ser `3.16` o superior.
+
+En una versión más antigua de Ubuntu (ejemplo: 16.04), puede seguir las instrucciones aquí para instalar la última versión de cmake. En debian necesitarás usar las fuentes de backports o construir Cmake manualmente.
```
Comment (Tony931023) [inline docs/es/linux-requirements.md:95]:
```suggestion
En una versión más antigua de Ubuntu (ejemplo: 16.04), puede seguir las instrucciones aquí para instalar la última versión de cmake. En Debian necesitarás usar las fuentes de backports o construir Cmake manualmente.
```

File: docs/es/linux-requirements.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+puedes instalar gcc-8: 
+
+```sh
+sudo apt-get install g++-8 gcc-8
+```
+
+---
+
```
Comment (Tony931023) [inline docs/es/linux-requirements.md:129]:
```suggestion
```

**Review by Tony931023 — CHANGES_REQUESTED** [review]
Update

**Review by Tony931023 — APPROVED** [review]

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
Thank you both @NicoC0 and @Tony931023!

**Pull request merged**